test/Login_reducers.test.js

```diff
@@ -0,0 +1,68 @@
+import loginReduser from './../app/src/redux/login';
+import * as loginActions from './../app/src/redux/login';
+import * as router from 'react-router';
+
+describe('loginReduser execution', () => {
```

Comment:
describe(this is not test)(after first describe no need empty line)
    describe (this is not test)
        it (this is test)
          (empty line after each test)
       it
  (empty line after each describe)
    describe
       it

       it


fix in this format